ketcher-autotests/tests/User-Interface/Select-Tools/select-tools.spec.ts

```diff
@@ -28,4 +36,18 @@ test.describe('Select tools tests', () => {
     const cursor = await page.getByTestId('canvas').getAttribute('cursor');
     expect(cursor).toBe('all-scroll');
   });
+
+  test('When the structure is pasted or dragged', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-8939
+      Select 'Benzene' from Templates
+      Paste it on canvas
+      Select 'Benzene' from Templates and paste over 'Benzene'
+      Place two 'Benzene' on the canvas and drag one onto the other
+    */
+    await drawBenzeneRing(page);
+    await selectDropdownTool(page, 'select-rectangle', 'select-fragment');
+    // eslint-disable-next-line no-magic-numbers
+    await clickOnAtom(page, 'C', 4);
```

Comment:
use CONST so you will not have to disable eslint (otherwise there's no point in eslint at all)

Reply:
Agree. And it is better give a meaningful name for variable:
```
const anyAtom = 4;
or
const atomWithQueryFeatures = 4;
```

Reply:
Done.